composer.json

```diff
@@ -50,10 +57,7 @@
         ]
     },
     "config": {
-        "bin-dir": "bin",
-        "platform": {
-            "php": "5.3.9"
```

Comment:
not good. We want to keep the demo compatible with older PHP versions, to ensure that someone running ``symfony demo`` gets a working demo

@javiereguiluz we may switch to 5.4 as min target though, as the installer itself requires 5.4 anyway.